Meetings/SG/20190903.md

```diff
@@ -0,0 +1,67 @@
+# ETF Steering Group (ETF SG) - 15th meeting
+
+## Logistics
+
+Location: Webex
+
+Date/Time: 03/09/2019, 10:00-11:30
+
+## Attendance
+
+- SG members:
+  - JRC: Michael Lutz (main), Marco Minghini
+  - ii: Clemens Portele (main), Jon Herrmann
+
+- Observers:
+  - Guadaltel: Carlos Palma
+
+## Minutes of previous meeting and open actions
+
+[Minutes by JRC](https://github.com/etf-validator/governance/blob/master/Meetings/SG/20190718.adoc) accepted and merged.

```

Comment:
the correct link is https://github.com/etf-validator/governance/blob/master/Meetings/SG/20190718.md

Reply:
Corrected.